.github/CONTRIBUTING.md

```diff
@@ -161,3 +162,34 @@ If you're new to open-source or to Briefer, you can start by looking at the issu
 When opening a pull request, please make sure to fill the template with as much information as you can. This will help us understand your contribution and review it more effectively.
 
 After submitting a pull request, please make sure to sign [our CLA (Contributor License Agreement)](./CLA.md) by following the instructions that our bot will provide you. This is a necessary step for us to accept your contribution.
+
+Here’s the fully formatted version of the content with properly labeled code blocks:
+
+<br/>
```

Comment:
Shouldn't we remove this? Not sure I understand why this is here

IMO removing is fine